ui/src/app/pages/data/criteria-search.tsx

```diff
@@ -299,20 +298,20 @@ export const CriteriaSearch = fp.flow(withUrlParams(), withCurrentWorkspace())(c
   }
 
   render() {
-    const {backFn, cohortContext, conceptSearchTerms, selectedSurvey, source} = this.props;
+    const {backFn, cohortContext, conceptSearchTerms} = this.props;
```

Comment:
nit: can we have cohortContext:{source, selectedSurvey} for readability?

Reply:
Done